supplements/tashas-cauldron-of-everything/archetypes/wizard-bladesinging.xml

```diff
@@ -0,0 +1,106 @@
+			<grant type="Archetype Feature" id="ID_WOTC_TCOE_ARCHETYPE_FEATURE_BLADESINGING_SONG_OF_VICTORY" level="14" />
+		</rules>
+	</element>
+	<element name="Training in War and Song" type="Archetype Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_ARCHETYPE_FEATURE_BLADESINGING_TRAINING_IN_WAR_AND_SONG">
+		<description>
+			<p><em>2nd-level Bladesinging feature</em></p>
+			<p>You gain proficiency with light armor, and you gain proficiency with one type of one-handed melee weapon of your choice.</p>
+			<p class="indent">You also gain proficiency in the Performance skill if you don’t already have it.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
+		<rules>
+			<grant type="Proficiency" id="ID_PROFICIENCY_ARMOR_PROFICIENCY_LIGHT_ARMOR" />
+			<select type="Proficiency" name="Weapon Proficiency, Training in War and Song" supports="(ID_INTERNAL_WEAPON_CATEGORY_SIMPLE_MELEE||ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_MELEE),!ID_INTERNAL_WEAPON_PROPERTY_TWOHANDED" />
+			<grant type="Proficiency" id="ID_PROFICIENCY_SKILL_PERFORMANCE" />
+		</rules>
+	</element>
+	<element name="Bladesong" type="Archetype Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_ARCHETYPE_FEATURE_BLADESINGING_BLADESONG">
+		<description>
+			<p><em>2nd-level Bladesinging feature</em></p>
+			<p>You can invoke an elven magic called the Bladesong, provided that you aren't wearing medium or heavy armor or using a shield. It graces you with supernatural speed, agility, and focus.</p>
+			<p class="indent">You can use a bonus action to start the Bladesong, which lasts for 1 minute. It ends early if you are incapacitated, if you don medium or heavy armor or a shield, or if you use two hands to make an attack with a weapon. You can also dismiss the Bladesong at any time (no action required).</p>
+			<p class="indent">While your Bladesong is active, you gain the following benefits:</p>
+			<ul>
+				<li>You gain a bonus to your AC equal to your Intelligence modifier (minimum of +1).</li>
+				<li>Your walking speed increases by 10 feet.</li>
+				<li>You have advantage on Dexterity (Acrobatics) checks.</li>
+				<li>You gain a bonus to any Concentration saving throw you make to maintain your concentration on a spell. The bonus equals your Intelligence modifier (minimum of +1).</li>
+			</ul>
+			<p>You can use this feature a number of times equal to your proficiency bonus, and you regain all expended uses of it when you finish a long rest.</p>
+		</description>
+		<sheet action="Bonus Action" usage="{{proficiency}}/Long Rest">
+			<description>You can {{arcana}} {{medicine}}start the Bladesong, which lasts for 1 minute. It ends early if you are incapacitated, if you don medium or heavy armor or a shield, or if you use two hands to make an attack with a weapon. You can also dismiss the Bladesong at any time.
```

Comment:
"You can {{arcana}} {{medicine}}start the Bladesong"

Looks like a copy paste oops